src/TodoList.jsx

```diff
@@ -1,19 +1,11 @@
 
 import TodoListItem from './TodoListItem'
-
-// 1- Above this App() function, I created an array with some initial data and store it in a variable named todoList.
-const todoList =[
-    {id: 1,title:"Complete assignment"},
-    {id: 2,title:"Attend at least one mentor session weekly"},
-    {id: 3,title:"Do more Project"}
-  ];
-function TodoList(){
-
+function TodoList({todoList,onRemoveTodo}){
     return (
             <ul>  
-              {
-                todoList.map((item)=>(
-                  <TodoListItem key={item.id} todo={item}/>
+              {               
```

Comment:
Here I suggest improving the syntax and readability of your TodoList component and use a more concise format which follows best practices. Instead of destructuring id and title directly in the map function parameter, we can use a single todo parameter. This makes the code a bit cleaner and easier to read.
```
    <ul>
      {todoList.map(todo => (
        <TodoListItem
          key={todo.id}
          id={todo.id}
          title={todo.title}
          onRemoveTodo={onRemoveTodo}
        />
      ))}
    </ul>
```